Framework/DataHandling/test/LoadILLTOF2Test.h

```diff
@@ -117,7 +127,7 @@ class LoadILLTOF2Test : public CxxTest::TestSuite {
     const size_t monitorCount = 1;
     MatrixWorkspace_sptr ws =
         loadDataFile("ILL/IN4/084446.nxs", histogramCount, monitorCount,
-                     channelCount, tofDelay, tofChannelWidth);
+                     channelCount, tofDelay, tofChannelWidth, true);
```

Comment:
For this and all of the other true/falses, it would be clearer if you defined some well-named variables/constants and used them instead.  It is not clear from reading the method call, what these true/falses are doing without knowing the method being called in detail.